x/farming/keeper/reward.go

```diff
@@ -471,7 +484,15 @@ func (k Keeper) AllocateRewards(ctx sdk.Context) error {
 
 	// For each staking coin denom in the table, increase cumulative unit rewards
 	// and increment current epoch number by 1.
-	for stakingCoinDenom, unitRewards := range unitRewardsByDenom {
+	// Sort keys for deterministic execution.
+	var denoms []string
+	for denom := range unitRewardsByDenom {
+		denoms = append(denoms, denom)
+	}
+	sort.Strings(denoms)
```

Comment:
Same comment as above goes here in this line.
```suggestion
	sort.Strings(denoms)
	
```

Reply:
I updated the code style, thanks.